src/gui/options.ui

```diff
@@ -2290,6 +2290,37 @@
              </widget>
             </item>
             <item>
+             <widget class="QGroupBox" name="checkEnableAddTrackers">
+              <property name="sizePolicy">
+               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+                <horstretch>0</horstretch>
+                <verstretch>0</verstretch>
+               </sizepolicy>
+              </property>
+              <property name="title">
+               <string>A&amp;utomatically add these trackers to new downloads:</string>
```

Comment:
Do other settings here have accelerators? If not, remove it.
Yes it will be merged. However, I am working from the top of the PR queue. Eventually I'll come to this.